src/templates/sysconfig.html

```diff
@@ -496,6 +496,10 @@ <h3 class='p-2 pl-3 section-title'>{{ 'Uploaded files configuration'|trans }}</h
     </div>
     <hr>
 
+    {% include 'binary-setting.html' with {'label': 'S3 verify SSL certificates (disable only for HTTP. This is insecure!)'|trans, 'slug': 's3_verify_cert'} %}
```

Comment:
I think it is best to re-use the existing translated string: "Verify TLS certificate".

When you configure S3 storage you probably know already that disabling cert verification is not good practice, so I don't think there is a need to add such a sentence.